monai/inferers/utils.py

```diff
@@ -142,7 +142,9 @@ def sliding_window_inference(
         diff = max(roi_size[k - 2] - inputs.shape[k], 0)
         half = diff // 2
         pad_size.extend([half, diff - half])
-    inputs = F.pad(inputs, pad=pad_size, mode=look_up_option(padding_mode, PytorchPadMode), value=cval)
+
+    if any(pad_size) > 0:
```

Comment:
any returns a boolean value
```suggestion
    if any(pad_size):
```